docs/content/014-guides/021-abstract-types.md

```diff
@@ -0,0 +1,472 @@
+   - The union type has defined `resolveType`
+   - The model type includes `__typename` property whose type is a string literal matching the GraphQL object name (case sensitive).
+   - The fields where the union type is used include `__typename` in the returned model data
+
+#### Multiple Strategies at Runtime
+
+It is possible to have all three strategies enabled and thus use at once. In that case the following runtime precedence rules apply. Using a strategy here means the implementations of others of lower priority are discarded.
+
+1. `resolveType`
+2. `__typename`
+3. `isTypeOf`
+
+> The default `resolveType` implementation is actually to apply the other strategies. If you're curious how that looks internally you can see the code [here](https://github.com/graphql/graphql-js/blob/cadcef85a21e35ec6df7229b88182a4a4ad5b23a/src/execution/execute.js#L1132-L1170).
+
+#### Multiple Strategies at Buildtime
+
+Nexus leverages TypeScript statically encode all the rules discussed in this guide. When you have enabled multiple strategies it means that when Nexus detects a faulty implementation, then static type errors will be raised in _all_ places where a fix _could_ be made, but _might not_ be made. In practice this means that some of the errors you'll see will be noise since only a subset are pointing toward the strategy you will ultimately want to use. This is one reason why Nexus considers having multiple strategies an advanced pattern. The static errors produced are likely to confuse newcomers, who cannot tell which ones stem from which strategy (or that there even are multiple strategies!).
+
+#### Picking a Strategy
+
+There is no right or wrong strategy to use. Use the one that you and/or your team agree upon. These are some questions you can ask yourself:
+
+| Questions                                                                             | If affirmative, then maybe                                                                        |
+| ------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- |
+| Is my schema large? Do I have many collaborators?                                     | Modular (`__typename`, `isTypeOf`)                                                                |
+| Is my schema simple? Do I develop alone?                                              | Centralized (`resolveType`)                                                                       |
+| Do I keep a discriminant property in the databse already?                             | `__typename` in your model layer if you have one                                                  |
+| Do I have objects that are part of multiple unions types and/or implement interfaces? | `__typename` or `isTypeOf` to avoid repeated logic across multiple `resolveType` implementations) |
```

Comment:
While you explain the different strategy very well, unless I'm mistaking, you do not explain how to pick one strategy using the global config: `features: { abstractTypes: { isTypeOf|resolveType|backingType?: boolean } }`